vars/karmaExecuteTests.groovy

```diff
@@ -89,6 +89,8 @@ void call(Map parameters = [:]) {
 
         utils.pushToSWA([step: STEP_NAME], config)
 
+        config.dockerEnvVars = config.dockerEnvVars ?: [:]
```

Comment:
Should be obsolete as we have default values in place.
```suggestion
```

Reply:
Are you sure? I remember that I tried without this and a test case was failing... let's try again...

Reply:
Guess I was wrong 😄